docs/source/developer_guide/development-workflow.md

```diff
@@ -42,3 +42,25 @@ You can check that the JupyterLab extension was successful installed with:
 ```bash
 jupyter labextension list
 ```
+
+### Incremental Development
+
+Elyra supports incremental development using `--watch`. This allows you to make code changes to
+front-end packages and see them without running `make install` again.
+
+After installation run the following to watch for code changes and rebuild automatically:
+```bash
+make watch
+```
+
+Then in a separate terminal using the same environment, start JupyterLab in watch mode:
```

Comment:
is it worth clarifying what you mean by _same environment_? i.e., same Conda or Python virtual environment

Reply:
I was trying to figure out how to word that without assuming they were using conda. would saying `python environment` make sense? Personally I never had a concept of separate environments in my dev until I started working on lab and installed conda.

Reply:
yes, i think `Python environment` makes sense

Reply:
updated